checkout-versions.yaml

```diff
@@ -5,4 +5,4 @@
 
 versions:
   ramble: f5970aa     # develop on 1/08/2025 (newer than 0.5.0 release)
-  spack: 0b3b49b      # develop-2024-06-28 (newer than 0.22.0 release)
+  spack: c6d4037      # v0.23.0 (2024-11-13)
```

Comment:
I pulled the spack update version into its own PR https://github.com/LLNL/benchpark/pull/623; lets test it with other benchmarks and merge that PR first.

Reply:
Yes you beat me to it. Needed a new spack version because it was looking for `Package 'c' not found.` when trying to install salmon.